.busted

```diff
@@ -1,6 +1,6 @@
 --- .busted
 --
--- Copyright (c) 2013 Snowplow Analytics Ltd. All rights reserved.
```

Comment:
Shouldn't we change it to a range of years such as "Copyright (c) 2013-2022"? That's what I see in other trackers but haven't checked all.